insights/parsers/transparent_hugepage.py

```diff
@@ -0,0 +1,67 @@
+"""
+transparent_hugepage sysfs settings
+===================================
+
+Module for parsing the sysfs settings for transparent_hugepage:
+
+ThpUseZeroPage - file ``/sys/kernel/mm/transparent_hugepage/use_zero_page``
+---------------------------------------------------------------------------
+
+Gets the contents of /sys/kernel/mm/transparent_hugepage/use_zero_page, which is either 0 or 1.
+
+Sample input::
+
+    0
+
+Examples:
+
+    >>> shared[ThpUseZeroPage].use_zero_page
+    0
+
+
+ThpEnabled - file ``/sys/kernel/mm/transparent_hugepage/enabled``
+-----------------------------------------------------------------
+
+Gets the contents of  /sys/kernel/mm/transparent_hugepage/enabled, which is something like
+`always [madvise] never` where the active value is in brackets.
+
+If no option is active (that should never happen), `get_opt()` will return None.
```

Comment:
Need to replace reference to `get_opt` with attribute.  You can add `active_option` to an *Attributes* section in your docstring to include it in the documentation.

Reply:
@bfahr Thank you pointing out my omission.